src/main/java/gg/uhc/uhc/modules/portals/EndModule.java

```diff
@@ -86,17 +88,19 @@ public void onDisable() {
         }
     }
 
-    @EventHandler(ignoreCancelled = true)
+    @EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
```

Comment:
I don't normally like using HIGHEST priority when were are potentially using setCancelled due to it making it harder to override in another plugin. Is there a reason for using HIGHEST specifically and not just HIGH to go past the regular NORMAL?